apps/frontend/Dockerfile

```
@@ -0,0 +1,18 @@
+FROM node:18-alpine
```

no reason to not use latest
```suggestion
FROM node:24-alpine
```